src/nvidia_rag/ingestor_server/main.py

```
@@ -729,11 +730,8 @@ async def status(task_id: str) -> dict[str, Any]:
                 )
                 return {"state": "FAILED", "result": status_and_result.get("result")}
             else:
-                logger.error(
-                    f"Unknown task state: {
-                        INGESTION_TASK_HANDLER.get_task_state(task_id)
-                    }"
-                )
+                task_state = INGESTION_TASK_HANDLER.get_task_status(task_id)
+                logger.error(f"Unknown task state: {task_state}")
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)